src/components/BooksPage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+import Header from './Header';
+import Book from './Book';
+import Form from './Form';
+
+const BooksPage = () => {
+  const books = [
+    {
+      author: 'Suzanne Collins',
+      title: 'The Hunger Games',
+      id: uuidv4(),
+    },
+    {
+      author: 'Me',
+      title: 'Best Book Ever',
+      id: uuidv4(),
+    },
+  ];
+
+  // const handleSubmit = (e) => {
+  //   e.preventDefault()
+  // }
+
+  return (
+    <div>
+      <Header />
```

Comment:
- **[OPTIONAL]** Instead of repeatedly importing this Header component across multiple components you can remove it from the components and import it in the App components once and for all. It will appear on all pages.